package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@launchpadlab/lp-subsection-generator",
-  "version": "6.0.3",
+  "version": "6.1.0",
```

Comment:
If you expand the next section of code hidden by Github, you'll see the `bin` subsection. I'm wondering if adding an additional line like:
```
"generate-ionic-subsection": "./index.js --template-type ionic"
```
would cause **both** commands to be available to a user. Then we can change the documentation for the ionic client template to instruct the user to use the updated command. Not sure how to test this though until these changes are merged and we go through the yarn install process to see what happens.

Reply:
The documentation seems to suggest that multiple "bin" commands would be accepted (https://docs.npmjs.com/cli/v7/configuring-npm/package-json#bin).

Can you try this locally while it is symlinked?

Reply:
@mwislek bump